src/apps/aave/positions.ts

```diff
@@ -65,26 +65,28 @@ const hook: PositionsHook = {
       args: [aaveAddresses.poolAddressesProvider],
     })
 
-    const [userReserveData, [reserveIncentiveData, _userIncentivesData]] =
-      address
-        ? await client.multicall({
-            contracts: [
-              {
-                address: aaveAddresses.uiPoolDataProvider,
-                abi: uiPoolDataProviderV3Abi,
-                functionName: 'getUserReservesData',
-                args: [aaveAddresses.poolAddressesProvider, address as Address],
-              },
-              {
-                address: aaveAddresses.uiIncentiveDataProvider,
-                abi: uiIncentiveDataProviderV3Abi,
-                functionName: 'getFullReservesIncentiveData',
-                args: [aaveAddresses.poolAddressesProvider, address as Address],
-              },
-            ],
-            allowFailure: false,
-          })
-        : [undefined, [undefined, undefined]]
+    const [
+      [userReserveData, _userEmodeCategoryId] = [],
+      [reserveIncentiveData, _userIncentivesData],
+    ] = address
+      ? await client.multicall({
+          contracts: [
+            {
+              address: aaveAddresses.uiPoolDataProvider,
+              abi: uiPoolDataProviderV3Abi,
+              functionName: 'getUserReservesData',
+              args: [aaveAddresses.poolAddressesProvider, address as Address],
+            },
+            {
+              address: aaveAddresses.uiIncentiveDataProvider,
+              abi: uiIncentiveDataProviderV3Abi,
+              functionName: 'getFullReservesIncentiveData',
+              args: [aaveAddresses.poolAddressesProvider, address as Address],
+            },
+          ],
+          allowFailure: false,
+        })
+      : [undefined, [undefined, undefined]]
```

Comment:
should this be 
```suggestion
      : [[undefined, undefined], [undefined, undefined]]
```
per the above comment? Think you can avoid the `=[]` too in that case

Reply:
thats what was tripping up my typescript!